notifier/tasks.py

```diff
@@ -36,13 +36,26 @@ def generate_and_send_digests(users, from_dt, to_dt, language=None):
     `from_dt` and `to_dt` are datetime objects representing the start and end
     of the time window for which to generate a digest.
     """
+
+    # TODO: Need to implement an end point in LMS to get this list
+    orgs_to_skip = ["WhartonOnlineProfessionalEd"]
     settings.LANGUAGE_CODE = language or settings.LANGUAGE_CODE or DEFAULT_LANGUAGE
     users_by_id = dict((str(u['id']), u) for u in users)
     msgs = []
+
+    for user in users:
+        skip_user = False
+        for course_id in user["course_info"].keys():
+            for org in orgs_to_skip:
+                if org in course_id:
+                    skip_user = True
+        if skip_user:
+            del users_by_id[str(user["id"])]
```

Comment:
If we do this, the user won't get a digest for _any_ course, which isn't what we want - we just don't want to send digests for Wharton courses.  I think the right place to do the filtering is in `generate_digest_content`.  Let me take a look at that for a bit and get back to you.